src/js/components/Menu/__tests__/Menu-test.js

```diff
@@ -140,12 +140,6 @@ describe('Menu', () => {
         />
       </Grommet>,
     );
-
-    const firstItem = getByText('Item 1');
```

Comment:
Any reason why we changed this test?

Reply:
Hmm, I think it was because it couldn't work for the CSS approach, since there is no "StyledBoxGap" in that case. I suspect it's a holdover from the first version that didn't have the Grommet `options` flag. Now that I look at it, I'm wondering if we need it though, since we already have the snapshot check.

Reply:
It appears to be the only test where we are doing a white-box selector on StyledBoxGap as a class.

Reply:
I'm fine with removing this section of the test since we have the snapshot to cover this.